velox/docs/monthly-updates/june-2022.rst

```diff
@@ -0,0 +1,61 @@
+********************
```

Comment:
I'm not sure if it is required, but usually these markup lines are matching the length of the text.

Reply:
It is not required. But I fixed it.